src/test/java/hw2/ex1/pages/HomePage.java

```diff
@@ -0,0 +1,289 @@
+package hw2.ex1.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class HomePage extends AbstractPage {
+
+    private WebDriver driver;
+
+    public HomePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    /**
+     * STEP #1: Open test site by URL
+     * DATA: https://jdi-testing.github.io/jdi-light/index.html
+     * EXPECTED RESULT: Test site is opened
+     */
+    @Override
+    public void openPage() {
+        driver.get(getPageProperty("homePageURL"));
+    }
+
+    /**
+     * STEP #2: Assert Browser title
+     * DATA: "Home Page"
+     * EXPECTED RESULT: Browser title equals "Home Page"
+     */
+    @Override
+    public boolean checkTitle() {
+        return driver.getTitle().equals("Home Page") ? true : false;
+    }
+
+    /**
+     * STEP #3: Perform login
+     * DATA: username: Roman, pass: Jdi1234
+     * EXPECTED RESULT: User is logged
+     */
+    public void login() {
+        String loginCaret = getPageProperty("loginCaret"); //xpath todo
+        waitForElementLocatedBy(driver, By.xpath(loginCaret));
+        driver.findElement(By.xpath(loginCaret)).click();
```

Comment:
waitForElementLocatedBy returns an element, so  you do not have to find it one more time
```suggestion
        String loginCaret = getPageProperty("loginCaret"); //xpath todo
        waitForElementLocatedBy(driver, By.xpath(loginCaret)).click();
```

same for the rest code